.github/workflows/test_on_push.yaml

```diff
@@ -41,25 +42,25 @@ jobs:
           TEST_DATABASE_USER: test
           TEST_DATABASE_PSWD: test-pswd
           TEST_DATABASE_NAME: test-jobbergate
-          POETRY_HTTP_BASIC_PYPICLOUD_USERNAME: admin
-          POETRY_HTTP_BASIC_PYPICLOUD_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
-        run: make qa
+        run: |
+          poetry env use "3.8"
```

Comment:
Poetry was using the Python version it was installed from, the one from the system. Here it is forced to use the one installed by `actions/setup-python`, like suggested by: https://github.com/actions/setup-python/issues/374#issuecomment-1088938718